_includes/checklist.md

```diff
@@ -51,7 +50,7 @@ Everything in this section needs to be completed before the project will be sche
         * [ ] Resolve any visible security issues, re-running the scan as needed
         * [ ] Add the issue-free scan report or [documentation about false positives](https://before-you-ship.18f.gov/security/dynamic-scanning/#caveats) to the project's ATO folder.
 - [ ] If this is a new system, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to the site.
-- [ ] Ensure the staging environment has sufficient capacity to withstand the testing.
+- [ ] Ensure the production environment has sufficient capacity to withstand the testing.
```

Comment:
Maybe we call this "(soon-to-be) production" also, to match line 41. Or call it the testing environment?